packages/components/src/components/typography/Heading/Heading.tsx

```diff
@@ -1,8 +1,11 @@
 import styled from 'styled-components';
 
 import { Color, TypographyStyle, typography } from '@trezor/theme';
-import { FrameProps, TransientFrameProps, withFrameProps } from '../../common/frameProps';
-import { makePropsTransient } from '../../../utils/transientProps';
+import { FrameProps, withFrameProps } from '../../common/frameProps';
+import { makePropsTransient, TransientProps } from '../../../utils/transientProps';
+
+export const allowedHeadingFrameProps: (keyof FrameProps)[] = ['margin'];
```

Comment:
Can we export some type for  `(keyof FrameProps)` I fear we may end up repeating this in other places.